src/metabase/metabot.clj

```diff
@@ -70,6 +71,7 @@
                              dispatch-token) varr}))]
     (fn dispatch*
       ([]
+        ;; @maz: I'm not sure how to translate strings like this with variable references
        (keys-description (format "Here's what I can %s:" verb) fn-map))
```

Comment:
You want `(tru "Here's what I can {0}:" verb)`

Reply:
Okay, so how about something like the one on line 79:
```
(format "I don't know how to %s `%s`.\n%s"`
```
Does that become this? 
```
(tru "I don't know how to {0} `{1}`.\n{2}"`
```